### ISSUE TITLE
Q: Scaling of blog post

### ISSUE THREAD

**Issue by forsco** [thiagorossener/jekflix-template#25]
Hi @thiagorossener,

I love the template, and was wondering if you could help me out. I'm only starting to learn Markdown and Jekyll so I don't know where to look.

Currently if you only have one blog post published it scales to the full width of the page in the browser, is it possible to remove this functionality so that it leaves the post at a size similar to if you have 4 or 8 blog posts like https://lambda.sx/Yuy.png ?

Thanks for your time.

**Comment by anaclumos** [comment]
Hello, I have tried to fix this issue too.

The issue happens here:
https://github.com/thiagorossener/jekflix-template/blob/1b0881e9db83e044b9c0df5ebccf9a8ac2be9cde/assets/css/main.css#L3

This file is minified, but if you expand the file, you will find the following section around `Line 786`.

```css
.box-item {
    -ms-flex: 1 0 300px;
    -webkit-box-flex: 1;
    flex: 1 0 300px;
    margin: 0 0 1.875rem;
    display: inline-block;
    min-height: 17.813rem;
    -webkit-transition: all .3s;
    transition: all .3s;
    position: relative;
    z-index: 1;
}
```

The lines including the `flex` keyword made the post gigantic, I believe.
I have removed the following lines

```css
-ms-flex: 1 0 300px;
-webkit-box-flex: 1;
flex: 1 0 300px;
```

and then added,

```css
max-width: 300px;
```

This fixed the issue for me.

**Comment by forsco** [comment]
Thanks, I’ll give this a shot when I get home!

**Comment by thiagorossener** [comment]
Hey guys, I'm sorry I took that long to reply. Thank you @forsco for your words.

Well, @anaclumos solution do solve the problem, however, depending on the screen resolution, the right side gets too much space on it.

So, I took this morning in order to study the issue and came up with [this solution](https://github.com/thiagorossener/jekflix-template/commit/00bee9f0ab13a37e0ee50d9342394edc32366f32).

With some calc we are able to fix the size of the leftover items for a range of screen widths, I considered a maximum width of 3840px (a 4K monitor).

Hope this help you guys, and thank you again for suggest this change.
